File: basic/1023.cpp
#include <iostream>
#include <string>
using namespace std;
int cnt[10];
int main() {
    bool isFirst = true;
    string ans = "";
    for (int i = 0; i < 10; ++i) {
        cin >> cnt[i];
        if (i && cnt[i] && isFirst) {
            ans += i+'0';
            --cnt[i];
            isFirst = false;
        }
    }
    for (int i = 0; i < 10; ++i) {
        for (int j = 0; j < cnt[i]; ++j)
            ans += i+'0';
    }
    cout<<ans<<endl;
    return 0;
}

File: basic/1023.py
cnt = [int(i) for i in input().split(" ")]
ans = "0"
for i in range(1, 10):
    if cnt[i] > 0:
        cnt[i] -= 1
        ans = str(i)
        break
for i in range(10):
    ans += str(i) * cnt[i]

print(ans)
